Linshuo/client/src/client.cpp

```diff
@@ -0,0 +1,116 @@
+#include <ros/ros.h>
+#include <ros/time.h>
+
+#include <client/time.h>
+#include <client/show.h>
+
+#include <signal.h>
+#include <unistd.h>
+
+#include <ctime>
+#include <chrono>
+#include <cstring>
+#include <iostream>
+
+using namespace std;
+using namespace std::chrono;
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Do not use namespace using-directives.  Use using-declarations instead.  [build/namespaces] [5]